bench/axis_opencode_native_fallback_test.go

```
@@ -0,0 +1,71 @@
+			defer group.Done()
+			body, err := json.Marshal(nativeFallbackRequest{Model: "root"})
+			if err != nil {
+				errs <- err
+				return
+			}
+			response, err := http.Post(server.URL, "application/json", bytes.NewReader(body))
+			if err != nil {
+				errs <- err
+				return
+			}
+			defer response.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `response.Body.Close` is not checked

(errcheck)